config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  devise_for :users, :controllers => { registrations: 'registrations' }
+  resources :posts
```

Comment:
- **_The application routes uses `resources :posts, only: [:new, :create, :index]` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))_**

Reply:
I updated the resources, however, I needed to add additional routes since our app has additional features such as show, edit, and delete.